docs/develop/host-specific-api-model.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Using the host-specific API model
+description: 'Learn about the promise-based API model for Excel, OneNote, and Word add-ins.'
+ms.date: 07/27/2020
+localization_priority: Normal
+---
+
+# Using the host-specific API model
+
+This article describes how to use the API model for building add-ins in Excel, Word, and OneNote. It introduces core concepts that are fundamental to using the promise-based APIs. Note that this model is not supported by Office 2013 clients. Use the [Common API model](office-javascript-api-object-model.md) to work with those Office versions. For full platform availability notes, see [Office Add-in host and platform availability](../overview/office-add-in-availability.md).
+
+> [!NOTE]
+> The examples in this page use the Excel JavaScriptAPIs, but the concepts equally apply to Excel, Word, and OneNote add-ins, as well as SharePoint-embedded Visio diagrams that use the Office JavaScript APIs.
+
+## Asynchronous nature of the promise-based APIs
+
+The web-based add-ins run inside a browser container. This container is embedded within the Office application on desktop-based platforms, such as Office on Windows, and runs inside an HTML iFrame in Office on the web. Enabling the Office.js API to interact synchronously with the Office host across all supported platforms is not feasible due to performance considerations. Therefore, the `sync()` API call in Office.js returns a [promise](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise) that is resolved when the Office application completes the requested read or write actions. Also, you can queue up multiple actions, such as setting properties or invoking methods, and run them as a batch of commands with a single call to `sync()`, rather than sending a separate request for each action. The following sections describe how to accomplish this using the `run()` and `sync()` APIs.
+
+## *.run function
+
+`Excel.run`, `Word.run`, and `OneNote.run` execute a function that specifies the actions to perform against Excel, Word, and OneNote. `*.run` automatically creates a request context that you can use to interact with Office objects. When `*.run` completes, a promise is resolved, and any objects that were allocated at runtime are automatically released.
```

Comment:
So I don't know what the team's thoughts are globally on this but do we write Promise or promise? Technically it is the caps, but I think the lower case makes more sense

Reply:
My rule of thumb is that whenever you're talking about the concept, it's lowercase. If you're talking about the code object, it's `Promise`. It's the same logic I use for range versus `Range`.